tests/model_selection/test_incremental.py

```diff
@@ -529,48 +530,47 @@ def test_high_performing_models_are_retained_with_patience(c, s, a, b):
 
 @gen_cluster(client=True)
 def test_same_params_with_random_state(c, s, a, b):
-    X, y = make_classification(n_samples=100, n_features=5, chunks=(10, 5))
-    model = SGDClassifier(tol=1e-3, penalty="elasticnet")
+    X, y = make_classification(n_samples=100, n_features=10, chunks=10, random_state=0)
```

Comment:
Why the change in `n_features`?

Reply:
Because this is a test to ensure determinism. Increasing the number of features so each chunk shape is `(10, 10)` will make each `partial_fit` call extremely unstable.

This isn't required, but does make this test a little stronger.